en/Modules/language-management.md

```diff
@@ -0,0 +1,170 @@
+
+````csharp
+Configure<AbpLocalizationOptions>(options =>
+{
+    options.Languages.Add(new LanguageInfo("en", "en", "English"));
+    //Add other languages here...
+});
+````
+
+`AbpLocalizationOptions` properties:
+
+* `Resources` : Represents list of resource files used for localization in the application.
+* `GlobalContributors` :  Global localization contributors. Global contributors have precedence over other contributors.
+* `Languages` : List of languages available in the application.
+
+## Internals
+
+### Domain Layer
+
+#### Aggregates
+
+This module follows the [Entity Best Practices & Conventions](https://docs.abp.io/en/abp/latest/Best-Practices/Entities) guide.
+
+##### Language
+
+* `Language` (aggregate root): Represents a language in the system.
+
+#### Repositories
+
+This module follows the [Repository Best Practices & Conventions](https://docs.abp.io/en/abp/latest/Best-Practices/Repositories) guide.
+
+Following custom repositories are defined for this module:
+
+* `ILanguageRepository`
+* `ILanguageTextRepository`
+
+#### Domain Services
+
+This module doesn't define any domain service.
+
+### Settings
+
+This module doesn't define any setting.
+
+### Application Layer
+
+#### Application Services
+
+* `LanguageAppService` (implements `ILanguageAppService`): Implements the use cases of the language management UI.
+* `LanguageTextAppService` (implements `ILanguageTextAppService`): Implements the use cases of the language texts management UI. 
+
+### Database Providers
+
+#### Common
+
+##### Table/Collection Prefix & Schema
+
+All tables/collections use the `Abp` prefix by default. Set static properties on the `LanguageManagementDbProperties` class if you need to change the table prefix or set a schema name (if supported by your database provider).
+
+##### Connection String
+
+This module uses `AbpLanguageManagement` for the connection string name. If you don't define a connection string with this name, it fallbacks to the `Default` connection string.
+
+See the [connection strings](https://docs.abp.io/en/abp/latest/Connection-Strings) documentation for details.
+
+#### Entity Framework Core
+
+##### Tables
+
+* **AbpLanguages**
+* **AbpLanguageTexts**
+
+#### MongoDB
+
+##### Collections
+
+* **AbpLanguages**
+* **AbpLanguageTexts**
+
+### Permissions
+
+See the `LanguageManagementPermissions` class members for all permissions defined for this module.
+
+## Distributed Events
+
+This module defines events for `Language` aggregate and `LanguageText` entity. When a `Language` changes, `LanguageListCacheInvalidator` invalidates the cache for `LanguageListCacheItem`. When a  `LanguageTextCacheItem ` changes,  `LanguageTextCacheInvalidator` invalidates the cache for  `LanguageTextCacheItem `. See the [standard distributed events](https://docs.abp.io/en/abp/latest/Distributed-Event-Bus) for more information about distributed events.
```

Comment:
It doesn't define any distributed event, but registers to the event to invalidate the cache. And it is not a distributed event, [it is local event](https://github.com/volosoft/volo/blob/dev/abp/language-management/src/Volo.Abp.LanguageManagement.Domain/Volo/Abp/LanguageManagement/LanguageListCacheInvalidator.cs#L9). None of the current modules define an event.